README.md

```
@@ -2,6 +2,24 @@
 
 Follow these steps to contribute a new blog post to our website. We appreciate your contribution and look forward to featuring your insights on [blog.reclaimprotocol.org](https://blog.reclaimprotocol.org)!
 
+A non-technical contributor's guide to pushing blogs :
+# Step 0 : Setup
+## Clone this repo locally (first time only)
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)